api/v2/routes/homepage.php

```diff
@@ -423,6 +423,15 @@
 		->withHeader('Content-Type', 'application/json;charset=UTF-8')
 		->withStatus($GLOBALS['responseCode']);
 });
+$app->post('/homepage/jackett/download/', function ($request, $response, $args) {
```

Comment:
The urlencoded url parameter did not play well as an argument. That's why I used post data instead.